src/components/Button/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Should we be replacing all of our usage of Button and IconButton with this new one?

Reply:
We can replace our usages of `Button` but not `IconButton` since it's not based from the Button component. I'll do the change.